src/lib/structures/game/Game.ts

```diff
@@ -204,7 +203,7 @@ export abstract class Game {
 					let content = `${userMention(player.id)}, thanks for playing! You listened to ${prefixAndPluralize(
 						'song',
 						player.songsListenedTo
-					)}, guessed ${score} of them correctly, `;
+					)}, scored ${score}, `;
```

Comment:
I feel like you're leaving this hanging without a description of some sort. Scored x what? We can't really say points because that's calculated a different way